web-www/package.json

```diff
@@ -59,6 +61,7 @@
     "start": "npm run develop",
     "serve": "gatsby serve",
     "clean": "gatsby clean",
-    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1",
+    "graphql:codegen": "graphql-codegen --config codegen.yml"
```

Comment:
Can we add one or two lines to the README about this?

Reply:
good call - done 👍